images/mcp230xx.svg

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="301px" height="61px" viewBox="0 0 301 61" enable-background="new 0 0 301 61" xml:space="preserve">  <image id="image0" width="301" height="61" x="0" y="0"
+    xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAS0AAAA9CAQAAABSS/dKAAAABGdBTUEAALGPC/xhBQAAACBjSFJN
+AAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAAAmJLR0QA/4ePzL8AAAAHdElN
```

Comment:
This is not an SVG - it is a PNG embedded in a SVG.

I can make a MCP230XX svg banner later if you'd like.

Reply:
That'd be ideal, thanks!